src/vm/jitinterface.cpp

```diff
@@ -12517,8 +12506,8 @@ CorJitResult CallCompileMethodWithSEHWrapper(EEJitManager *jitMgr,
         flags.Set(CORJIT_FLAGS::CORJIT_FLAG_FRAMED);
     if (g_pConfig->JitAlignLoops())
         flags.Set(CORJIT_FLAGS::CORJIT_FLAG_ALIGN_LOOPS);
-    if (ftn->IsVersionableWithJumpStamp() || g_pConfig->AddRejitNops())
-        flags.Set(CORJIT_FLAGS::CORJIT_FLAG_PROF_REJIT_NOPS);
+//    if (ftn->IsVersionableWithJumpStamp() || g_pConfig->AddRejitNops())
+//        flags.Set(CORJIT_FLAGS::CORJIT_FLAG_PROF_REJIT_NOPS);
```

Comment:
Commented out rather than deleted?

Reply:
good catch - this change was not supposed to be in this delta